src/UniswapWormholeMessageSender.sol

```diff
@@ -19,13 +19,17 @@ interface IWormhole {
     function messageFee() external view returns (uint256);
 }
 
+bytes32 constant messagePayloadVersion = keccak256(abi.encode("UniswapWormholeMessageSenderv1 (bytes32 receivedMessagePayloadVersion, address[] memory targets, uint256[] memory values, bytes[] memory datas, address messageReceiver, uint16 receiverChainId)"));
+
+function generateMessagePayload(address[] memory _targets, uint256[] memory _values, bytes[] memory _calldatas, address _messageReceiver, uint16 _receiverChainId) pure returns(bytes memory) {
```

Comment:
This function should be declared as `internal`.

Reply:
It's a free function and as such doesn't have visibility. I made it a free function such that the test code can call it. 

I see a few alternatives, but not sure what the best practices in Solidity are: 
1. Make it part of the contract but then it either needs to be public/external(?) such that the test code can call it
2. the test needs to inherit UniswapWormholeMessageSender, which is a pattern I've seen in other places, but seems weird to me. 
3. Create another contract with only that method and have the Test as well as UniswapWormholeMessageSender inherit it. Seems like the cleanest method, but probably overkill.

Reply:
I'll keep it as is for now.